low-level/sensors/impl/generic/node_hw_alerts_info.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2001-2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+from collections import namedtuple
+from framework.base.sspl_constants import DEFAULT_RECOMMENDATION
+
+Alert = namedtuple('AlertInfo',
+                   ['type', 'severity', 'description',
+                    'impact', 'recommendation'])
+
+alert_for_event = {
+    "Drive Slot / Bay": {
+        "Drive Present": {
+            "Asserted": Alert(
+                    "insertion", "informational",
+                    "Disk in slot '{}' [{}] is inserted/added.",
+                    # Disk in slot '0' [HDD 0 (0xF1)] is inserted/added.
+                    "None", "None"),
+            "Deasserted": Alert(
+                    "missing", "critical",
+                    "Disk in slot '{}' [{}] is missing/removed.",
+                    "Disk '{}' is not available.",
+                    DEFAULT_RECOMMENDATION)
+        },
+        "Drive Fault": {
+            "Asserted": Alert(
+                    "fault", "critial",
+                    "Disk in slot '{}' [{}] is faulty.",
+                    "Disk '{}' is not usable.",
```

Comment:
Update Impact  to
Disk functioning may get impacted if left unaddressed

Reply:
addressed.